MOBILE_CLIENT/IOS/_HARDENING/ATS_MALFORMATTED/meta.json

```diff
@@ -0,0 +1,23 @@
+{
+  "risk_rating": "hardening",
+  "short_description": "The application uses a malformed NSExceptionDomains structure.",
+  "references": {
+    "Custom App Transport Security (ATS) configurations for named domains.": "https://developer.apple.com/documentation/bundleresources/information_property_list/nsapptransportsecurity/nsexceptiondomains"
+  },
+  "title": "Non-Conformity with Expected Structure for NSExceptionDomains",
```

Comment:
This is long, maybe use somthing precise like "Malformed Plist File" !

Reply:
sure